c2corg_ui/static/partials/cards/articles.html

```diff
@@ -0,0 +1,20 @@
+<a ng-href="{{cardCtrl.createURL()}}" ng-init="doc = cardCtrl.doc; locale = cardCtrl.locale;">
+  <div class="outing-author text-center">
```

Comment:
it is, as is strange using routes-outings for preferences :P we should have some generic `document-author`, `document-outings`, but it won't forcibly be the same. Maybe we can open an issue along the lines of the issue where we want to rename "route_" functions that have nothing to do with route.